HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -116,142 +126,79 @@ public void onConnected() {
 	}
 
 	public void onError(Exception e) {
-		Toast.makeText(
-                MainActivity.this, 
-                e.getMessage(), 
-                Toast.LENGTH_LONG).show();
+		Toast.makeText(MainActivity.this, e.getMessage(), Toast.LENGTH_LONG).show();
 	}
 
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item) {
-	    switch (item.getItemId()) {
-	        case R.id.action_connect:
-	            doConnect();
-	            return true;
-	        default:
-	            return super.onOptionsItemSelected(item);
-	    }
+		switch (item.getItemId()) {
+			case R.id.action_connect:
+				doConnect();
+				return true;
+			default:
+				return super.onOptionsItemSelected(item);
+		}
 	}
-	
+
 	private void doConnect() {
-		if (!service.isAppConnected()) {
-			connectProgressDialog = ProgressDialog.show(
-                    MainActivity.this,
-                    "",
-                    "Please wait...", 
-                    true);
+		if (!mService.isAppConnected()) {
+			connectProgressDialog = ProgressDialog.show(MainActivity.this, "", "Please wait...", true);
```

Comment:
>"" [](start = 66, length = 2)

Is there an equivalent of string.Empty in Java? If there is, we should use that constant instead of "", although I am not sure if that is a style rule in java.